app/controllers/offerings_controller.rb

```diff
@@ -0,0 +1,30 @@
+class OfferingsController < ApplicationController
+
+  #show all offers; we don't use this but page exists
+  def index
+    @offers = Offering.all
+  end
+
+  def new
+    @offer = Offering.new
+    @boats = Boat.all
```

Comment:
we use boats.index for that.




Met vriendelijke groet,
Kind regards,

Iris Bune

On Wed, Feb 10, 2016 at 10:43 AM, ChielHackman <notifications@github.com>
wrote:

> In app/controllers/offerings_controller.rb
> <https://github.com/navase/boatconnection/pull/14#discussion_r52434700>:
>
> > @@ -0,0 +1,30 @@
> > +class OfferingsController < ApplicationController
> > +
> > +  #show all offers; we don't use this but page exists
> > +  def index
> > +    @offers = Offering.all
> > +  end
> > +
> > +  def new
> > +    @offer = Offering.new
> > +    @boats = Boat.all
>
> #show all offers; we don't use this but page exists
>
> anyway, when you add after you added a new offering you will get a list of
> all boats i guess?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/navase/boatconnection/pull/14/files#r52434700>.
>